modular_chomp/code/modules/mining/shelter_atoms_ch.dm

```diff
@@ -181,12 +181,13 @@
 	..()
 
 // Allows resetting the capsule if the wrong template is chosen.
-/obj/item/device/survivalcapsule/superpose/attackby(obj/item/W, mob/user)
-	if(istype(W, /obj/item/weapon/pen) && !used)
-		template_id = null
-		template = null // Important to reset both, otherwise the template cannot be reset once the pod has been deployed.
-		to_chat(user, "<span class='notice'>You reset the pod's selection.</span>")
-	..()
+/obj/item/device/survivalcapsule/superpose/verb/resetpod()
+	set name = "Reset Active Pod"
```

Comment:
sanity of it having being used as been stripped. Is the object directly deleted on use?